### PR TITLE
Part 1: Hello world

### PR DESCRIPTION
**Pull request by ltxitai**
Add Foundations, configure project, conform to LT style guide.
Show "Hello, World!" label, positioned using Masonry.

### CODE REVIEW

File: Baygur/BGRAppDelegate.h [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2019 Lightricks. All rights reserved.
+// Created by Itai Shaked.
+
+/// Used to initialize and run the application UI.
+@interface BGRAppDelegate : UIResponder <UIApplicationDelegate>
+
+/// Main window for the application. 
+@property (strong, nonatomic) UIWindow *window;
```
Comment (gershonho) [inline Baygur/BGRAppDelegate.h]: `readonly, nonatomic`

File: Baygur/BGRViewController.mm [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+@interface BGRViewController ()
+
+/// Label to show text in the view.
+@property (readwrite, nonatomic) UILabel *helloWorldLabel;
+
+@end
+
+@implementation BGRViewController
+
+#pragma mark - UIViewController -
```
Comment (gershonho) [inline Baygur/BGRViewController.mm]: Should be 3 pragmas

File: Baygur/BGRViewController.mm [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2019 Lightricks. All rights reserved.
+// Created by Itai Shaked.
+
+#import "BGRViewController.h"
+
+@interface BGRViewController ()
+
+/// Label to show text in the view.
+@property (readwrite, nonatomic) UILabel *helloWorldLabel;
+
```
Comment (gershonho) [inline Baygur/BGRViewController.mm]: Make readonly. Only properties that can be changed after initialisation must be readwrite.

**Comment by ltxitai** [comment]
Round.

**Comment by ltxitai** [comment]
Done.

**Pull request merged**